internal/git/commits.go

```diff
@@ -46,6 +51,18 @@ func GetCommits(start, end time.Time) ([]Commit, error) {
 	}
 
 	commits = filterOutBots(commits)
+
+	// TODO we could be smarter about loading/compiling the regular expressions just once.
```

Comment:
Yeah, I should probably extract a lot of this logic into something else that we can just pass a config into, but this is hardly the least performant part of this code.